---
layout: page
title: Publications
permalink: /publicatons/
---

\[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/?term=welch%2C+joshua+d%5Bauthor%5D)\] \[[Google Scholar](https://scholar.google.com/citations?user=XQ7bqCMAAAAJ&hl=en)\]

Welch lab members are indicated in **bold**.

**Kriebel AR**, **Welch JD**. Nonnegative matrix factorization integrates single-cell multi-omic datasets with partially overlapping features. bioRxiv. \[[HTML](https://www.biorxiv.org/content/10.1101/2021.04.09.439160v1)\]

Matsushita Y, **Ka Yan Chu A**, Ono W, **Welch JD**, Ono N. Intercellular interactions of an adipogenic CXCL12‐expressing stromal cell subset in murine bone marrow. Journal of Bone and Mineral Research 2021. https://doi.org/10.1002/jbmr.4282 \[[HTML](https://asbmr.onlinelibrary.wiley.com/doi/10.1002/jbmr.4282)\]

**Yu H**, **Welch JD**. Sampling from disentangled representations of single-cell data using generative adversarial networks. Genome Biology 22, 158 (2021). https://doi.org/10.1186/s13059-021-02373-4. Also presented at Learning Meaningful Representations of Life (LMRL 2020), Workshop at NeurIPS 2020. \[[PDF](https://genomebiology.biomedcentral.com/track/pdf/10.1186/s13059-021-02373-4.pdf)\]

**Bao Y**, **Welch JD**. Real-Time, Direct Classification of Nanopore Signals Using SquiggleNet. bioRxiv. Also presented at Machine Learning in Computational Biology (MLCB 2020). https://www.biorxiv.org/content/10.1101/2021.01.15.426907v1

BRAIN Initiative Cell Census Network (BICCN), Ricky S. Adkins, Andrew I. Aldridge, ..., **Joshua D. Welch**, Owen White, Elora Williams, Fangming Xie, Peng Xie, Feng Xiong, X. William Yang, Anna Marie Yanny, Zizhen Yao, Lulu Yin, Yang Yu, Jing Yuan, Hongkui Zeng, Kun Zhang, Meng Zhang, Zhuzhu Zhang, Sujun Zhao, Xuan Zhao, Jingtian Zhou, Xiaowei Zhuang, Brian Zingg. A multimodal cell census and atlas of the mammalian primary motor cortex. bioRxiv. https://doi.org/10.1101/2020.10.19.343129 \[[PDF](https://www.biorxiv.org/content/10.1101/2020.10.19.343129v1.full.pdf)\]

Alghamri MS, Avvari RP, Thalla R, Kamran N, Zhang L, Ventosa M, Taher A, Faisal SM, Núñez FJ, Fabiani MBG, Haase S, Carney S, Orringer D, Hervey-Jumper S, Heth J, Patil PG, Al-Holou WN, Eddy K, Merajver S, Ulintz PJ, **Welch JD**, **Gao C**, **Liu J**, Núñez G, Hambardzumyan D, Lowenstein PR, Castro M. G-CSF Secreted by Epigenetically Reprogrammed Mutant IDH1 Glioma Stem Cells Reverses the Meloid Cells'-Mediated Immunosuppressive Tumor Microenvironment. bioRxiv. https://doi.org/10.1101/2020.07.22.215954 \[[PDF](https://www.biorxiv.org/content/biorxiv/early/2020/07/24/2020.07.22.215954.full.pdf?%3Fcollection=)\]

Zizhen Yao, Hanqing Liu, Fangming Xie, Stephan Fischer, ..., Bosiljka Tasic,
**Joshua D. Welch**, Joseph R. Ecker, Evan Z Macosko, Bing Ren, BRAIN Initiative Cell Census Network
(BICCN), Hongkui Zeng, Eran A. Mukamel. An integrated transcriptomic and epigenomic atlas of mouse primary motor cortex cell types. bioRxiv. https://doi.org/10.1101/2020.02.29.970558 \[[PDF](https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.970558.full.pdf)\]

**Gao C**, **Liu J**, **Kriebel AR**, Preissl S, Luo C, Castanon R, Sandoval J, Rivkin A, Nery JR, Behrens MM, Ecker JR, Ren B, **Welch JD**. Iterative Single-Cell Multi-Omic Integration Using Online Learning. Nature Biotechnology (2021). https://doi.org/10.1038/s41587-021-00867-x. Also accepted for presentation at RECOMB 2020. \[[HTML](https://www.nature.com/articles/s41587-021-00867-x)\]


**Liu J**\*, **Gao C**\*, **Sodicoff J**, Kozareva V, Macosko EZ, **Welch JD**. Jointly defining cell types from multiple single-cell datasets using LIGER. Nature Protocols (2020). https://doi.org/10.1038/s41596-020-0391-8 \[[PDF](https://rdcu.be/b8pqb)\] \*Equal contribution

Matsushita Y, Nagata M, Kozloff K, **Welch J**, Mizuhashi K, Tokavanich N, Hallett S, Link D, Nagasawa T, Ono W, and Ono N. A Wnt-mediated transformation of the bone marrow stromal cell identity orchestrates skeletal regeneration. Nature Communications. 11, 332 (2020). \[[HTML](https://www.nature.com/articles/s41467-019-14029-w)\] \[[PDF](https://www.nature.com/articles/s41467-019-14029-w.pdf)\]

Zhou Y, Liu Z, **Welch JD**, Gao X, Wang L, Garbutt T, Keepers B, Ma H, Prins JF, Shen W, Liu J, Qian L. Single-Cell Transcriptomic Analyses of Cell Fate Transitions during Human Cardiac Reprogramming. Cell Stem Cell. 2019 Jun 12. pii: S1934-5909(19)30224-3. doi: 10.1016/j.stem.2019.05.020. \[[HTML](https://www.sciencedirect.com/science/article/pii/S1934590919302243)\]

**Welch JD**\*, Kozareva V, Ferreira A, Vanderburg C, Martin C, Macosko EZ\*. Single-cell multi-omic integration compares and contrasts features of brain cell identity. Cell. 2019 Jun 13;177(7):1873-1887.e17. doi: 10.1016/j.cell.2019.05.006. Epub 2019 Jun 6. \[[PDF](https://www.cell.com/action/showPdf?pii=S0092-8674%2819%2930504-5)\]  
\*Co-corresponding 

Rodriques SG\*, Stickels RR\*, Goeva A Martin CA, Murray E, Vanderburg CR, **Welch JD**, Chen LM, Chen F+,
Macosko EZ+. Slide-seq: A Scalable Technology for Measuring Genome-Wide Expression at High Spatial Resolution. Science 29 Mar 2019:
Vol. 363, Issue 6434, pp. 1463-1467 DOI: 10.1126/science.aaw1219 \[[PDF](http://science.sciencemag.org/content/sci/363/6434/1463.full.pdf)\]
\*Equal contribution
+Co-corresponding

Liu Z\*, Wang L\*, **Welch JD**\*, Ma H, Zhou Y, Vaseghi HR, Yu S, Wall JB, Alimohamadi S, Zheng M, Yin C, Shen W, Prins JF, Liu J, Qian L. Single-cell transcriptomics reconstructs fate conversion from fibroblast to cardiomyocyte. Nature (02 November 2017). 551, 100–104 doi:10.1038/nature24454 \[[PDF](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5954984/pdf/nihms908332.pdf)\]
\*Equal contribution

**Welch JD**, Hartemink AJ, Prins JF. MATCHER: manifold alignment reveals correspondence between single cell transcriptome and epigenome dynamics. Genome Biology 2017. 18:138. \[[PDF](https://genomebiology.biomedcentral.com/track/pdf/10.1186/s13059-017-1269-0)\]
*Also accepted at RECOMB 2017 under the title “E Pluribus Unum: United States of Single Cells”.

Lackey P, **Welch JD**, Marzluff WM. TUT7 Catalyzes the Uridylation of the 3’ End of Histone mRNA for Rapid Degradation. RNA 2016. doi:10.1261/rna.058107.116 \[[PDF](https://rnajournal.cshlp.org/content/22/11/1673.full.pdf+html)\]

**Welch JD**\*, Williams LA\*, DiSalvo M\*, Brandt AT, Marayati R, Sims CE, Allbritton NL, Prins JF, Yeh JJ, Jones CD. Selective Single Cell Isolation for Genomics Using Microraft Arrays. Nucleic Acids Research 2016. doi: 10.1093/nar/gkw700 \[[PDF](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5041489/pdf/gkw700.pdf)\]
*Equal contribution

Lyons SM, Cunningham CH, **Welch JD**, Groh B, Guo AY, Wei B, Whitfield ML, Xiong Y, Marzluff WF. A subset of replication-dependent histone mRNAs are expressed as polyadenylated RNAs in terminally differentiated tissues. Nucleic Acids Research 2016. doi: 10.1093/nar/gkw620 \[[PDF](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5100578/pdf/gkw620.pdf)\]

**Welch JD**, Hartemink AJ, Prins JF. SLICER: Inferring branched, nonlinear cellular trajectories from single cell RNA-seq data. Genome Biology 2016, 17:106  doi:10.1186/s13059-016-0975-3 \[[PDF](https://genomebiology.biomedcentral.com/track/pdf/10.1186/s13059-016-0975-3)\]
*Also accepted at RECOMB 2016 (doi: http://dx.doi.org/10.1101/047845)

**Welch JD**, Hu Y, Prins JF. Robust Detection of Alternative Splicing in a Population of Single Cells. Nucleic Acids Research 2016. doi: 10.1093/nar/gkv1525 \[[PDF](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4856971/pdf/gkv1525.pdf)\]
*Also accepted as a highlight at ISMB 2016

Brooks L, Lyons SM, Mahoney JM, **Welch JD**, Liu Z, Marzluff WM, Whitfield ML. A multi-protein occupancy map of the mRNP on the 3′ end of histone mRNAs. RNA 2015. \[[PDF](https://rnajournal.cshlp.org/content/21/11/1943.full.pdf+html)\]

**Welch JD**\*, Slevin MK\*, Tatomer D, Duronio RJ, Prins JF, Marzluff WF. EnD-Seq and AppEnD: Sequencing 3′ ends to identify nontemplated tails and degradation intermediates. RNA 2015. \[[PDF](https://rnajournal.cshlp.org/content/21/7/1375.full.pdf+html)\]
*Equal contribution

**Welch JD**, Baran-Gale J, Perou C, Sethupathy P, Prins JF. Pseudogenes Transcribed in Breast Invasive Carcinoma Show Subtype-Specific Expression and ceRNA Potential. BMC Genomics 2015. \[[PDF](https://bmcgenomics.biomedcentral.com/track/pdf/10.1186/s12864-015-1227-8)\]

Slevin MK, Meaux S, **Welch JD**, Bigler R, Miliani de Marval PL, Su W, Rhoads RE, Prins JF, Marzluff WF Deep Sequencing Shows Multiple Oligouridylations Are Required for 3′ to 5′ Degradation of Histone mRNAs on Polyribosomes. Molecular cell 53 (6), 1020-1030. March 2014. \[[PDF](https://www.cell.com/action/showPdf?pii=S1097-2765%2814%2900199-3)\]

Lichtenberg J, Kurz K, Liang X, Al-ouran R, Neiman L, Nau LJ, **Welch JD,** Jacox E, Bitterman T, Ecker K, Elnitski L, Drews F, Lee SS, Welch LR. WordSeeker: concurrent bioinformatics software for discovering genome-wide patterns and
word-based genomic signatures. BMC Bioinformatics. 2010 Dec 21;11 Suppl 12:S6.

Lichtenberg J, Yilmaz A, **Welch JD**, Kurz K, Liang X, Drews F, Ecker K, Lee SS, Geisler M, Grotewold E, Welch LR. The word landscape of the non-coding segments of the Arabidopsis thaliana genome. BMC Genomics. 2009 Oct 8;10:463.

Lichtenberg J, Jacox E, **Welch JD**, Kurz K, Liang X, Yang MQ, Drews F, Ecker K, Lee SS, Elnitski L, Welch LR. Word-based characterization of promoters involved in human DNA repair pathways. BMC Genomics. 2009 Jul 7;10 Suppl 1:S18.
